docs/advanced/longhorn-v2.md

```diff
@@ -14,7 +14,7 @@ The Longhorn V2 Data Engine harnesses the power of the Storage Performance Devel
 
 :::caution
 
-The Longhorn V2 Data Engine is an **Experimental** feature in Harvester v1.4.0 and should not be utilized in a production environment.
+The Longhorn V2 Data Engine is an **Experimental** feature in Harvester v1.5 and should not be utilized in a production environment.
```

Comment:
```suggestion
The Longhorn V2 Data Engine is an **Experimental** feature and should not be utilized in a production environment.
```

Since the doc is already versioned, no point of maintaining this for every release.

Reply:
Done